plugins/modules/identity/keycloak/keycloak_group_rolemapping.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Marius Huysamen (@mhuysamen)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_group_rolemapping
+
+short_description: Allows administration of Keycloak group_rolemapping with the Keycloak API
+
+version_added: 5.7.0
```

Comment:
5.7.0 has been released, the next release is 5.8.0:
```suggestion
version_added: 5.8.0
```

Reply:
Noted.